### PR TITLE
Added second bin

### PR DESCRIPTION
**Pull request by YounesB-McGill**
Added second bin to XML Layout and Java Wi-Fi parameter example.

### CODE REVIEW

**Review by ajungmoon — APPROVED** [review]
Looks good to me, but I have not tested. Would you please post after testing & documenting? Thanks so much for working on it so quickly.

**Comment by YounesB-McGill** [comment]
I updated the documentation and tested it on the TA EV3, and it runs as expected. If no one has any objections I'll release this later tonight.

@SamCleland034 @TheGreatHegazi

File: README.md [hunk 1]
```diff
@@ -134,7 +135,37 @@ you can bypass this by changing this line (in `Resources`):
...
-  Do this only for the beta demo, and remember to change it back afterwards. 
+  Do this only for the beta demo, and remember to change it back afterwards.
+
+#### &nbsp;&nbsp;&nbsp;**• For the final competition:**
+  You must change your code to receive
+`redBin` and `greenBin` parameters instead of `bin`. To do that:
+  1. Update the Server XML layout description file ([`layout.xml`](Server/layout.xml))
+  and the example file ([`example_data_fill.xml`](Server/example_data_fill.xml))
+  by pulling from this repo or redownloading the latest version from MyCourses.
+  If you had any custom XML layout data files, update them to include the two bins.
```
Comment (mcgill-dpm) [inline README.md]: I added this clarification, since students may have forgotten that they need to update the server files.

Comment (YounesB-McGill) [inline README.md:146]: I added this clarification, since students may have forgotten that they need to update the server files.

**Comment by YounesB-McGill** [comment]
The force pushes above are to clean up redundant commits. I fixed a link in the student documentation.

**Pull request merged**